### PR TITLE
Update the license script for the Garnet tree

### PR DESCRIPTION
**Pull request by jason-simmons**
See https://github.com/flutter/engine/pull/4043

### CODE REVIEW

**Comment by jason-simmons** [comment]
@Hixie @chinmaygarde

File: travis/licenses.sh [hunk 1]
```diff
@@ -1,3 +1,4 @@
+set -ex
```
Comment (cbracken) [inline travis/licenses.sh]: Was the `x` intentional?

**Comment by cbracken** [comment]
lgtm

File: travis/licenses.sh (hunk 1 above)
Comment (jason-simmons) [reply travis/licenses.sh]: This is for consistency with some of the other Travis scripts (and possibly helpfui for debugging)

File: travis/licenses_golden/licenses_garnet [hunk 2]
```diff
@@ -0,0 +1,1242 @@
...
+of the copyright holder.
+====================================================================================================
+
+====================================================================================================
+LIBRARY: mozilla
+ORIGIN: ../../../garnet/public/lib/url/third_party/mozilla/url_parse.cc
+TYPE: LicenseType.lgpl
+FILE: ../../../garnet/public/lib/url/third_party/mozilla/url_parse.cc
+----------------------------------------------------------------------------------------------------
+GNU LESSER GENERAL PUBLIC LICENSE
```
Comment (Hixie) [inline travis/licenses_golden/licenses_garnet]: cc @jamesr Are you aware of this LGPL code?

**Comment by abarth** [comment]
You might want to whitelist the (handful) of directories you actually use in this repo.

**Comment by jason-simmons** [comment]
Removed the set -x (it was too verbose) and filtered garnet/public/lib/url out of the license crawl

**Pull request merged**